pkgs/development/libraries/harfbuzz/default.nix

```diff
@@ -34,15 +35,17 @@ stdenv.mkDerivation {
 
   configureFlags = [
     # not auto-detected by default
+    "--with-gobject=yes"
     "--with-graphite2=${if withGraphite2 then "yes" else "no"}"
     "--with-icu=${if withIcu then "yes" else "no"}"
-  ]
-    ++ stdenv.lib.optional withCoreText "--with-coretext=yes";
+  ] ++ stdenv.lib.optional withCoreText "--with-coretext=yes";
 
   nativeBuildInputs = [ pkgconfig libintl ];
 
-  buildInputs = [ glib freetype cairo ] # recommended by upstream
-    ++ stdenv.lib.optionals withCoreText [ ApplicationServices CoreText ];
+  buildInputs =
+    [ glib freetype cairo  # recommended by upstream
+      gobject-introspection
```

Comment:
This should likely go to `nativeBuildInputs`.

Reply:
Fixed, thanks.